src/app/lib/net.sh

```diff
@@ -19,15 +19,15 @@ net::configure_hosts() {
 	local ip=$(net::bridge_ip)
 
 	for host in "${hosts[@]}"; do
-          if ! hostess has $host > /dev/null ; then
+					hostonly=$(echo $host | awk -F/ '{print $1}')
+          if ! hostess has $hostonly > /dev/null ; then
                 if [ ! -z "$HORDE_DEBUG" ]; then
-                        io::err "sudo hostess add $host $ip"
+                        io::err "sudo hostess add $hostonly $ip"
                 fi
-                if ! sudo hostess add $host $ip > /dev/null ; then
+                if ! sudo hostess add $hostonly $ip > /dev/null ; then
                         io::err "problem configuring hostname '${host}'"
```

Comment:
Can you update the error message to report `hostonly`

Reply:
I considered that, but the reason I left it was reporting `$host` was so that in the case having multiple `host.com/endpoints` , you can immediately know which config out of the possible matches to `host.com` failed.

In my eyes, the added information for debugging is worth keeping the full config path in the output here.

If you disagree with that reasoning, of course I'll change it.